src/prefect/server/schemas/core.py

```diff
@@ -298,6 +298,11 @@ class FlowRun(ORMBaseModel):
     )
     # parent_task_run: "TaskRun" = None
 
+    job_variables: Optional[Dict[str, Any]] = Field(
```

Comment:
if we're doing `Dict[str, Any]` here, should we do that everywhere?

Reply:
I think I'd just pushed a change here making this `Optional[Dict...]` -- that should keep it in line with other places it's defined now.

Reply:
Is it more accurate for this to be `Dict[str, Any]` since the server default is `{}` and this a `default_factory=dict` here? Is there a scenario where this would be `None`?

Reply:
This model is used as a basis for our `FlowRunUpdate` inputs, and in that case we want to differentiate between users passing an update of `job_variables={}` vs not providing `job_variables` at all (`job_variables=None`). 

I'll update this to `default=None` because using `default_factory=dict` is actually a subtle bug in behavior. This is getting [patched in cloud too](https://github.com/PrefectHQ/nebula/pull/7183/).